code/workspaces/infrastructure-api/resources/rstudio.configmap.template.yml

```diff
@@ -4,5 +4,4 @@ kind: ConfigMap
 metadata:
   name: {{ configMapName }}
 data:
-  config: |
-    www-root-path={{{ basePath }}}
+  X-RStudio-Root-Path {{{ basePath }}}
```

Comment:
Just having looked at some more examples I think the `:`  is generally expected, if not explicitly required so may be best to add it.

Reply:
Agreed, fixed